suites/pacific/rados/sanity_rados.yaml

```diff
@@ -64,19 +85,59 @@ tests:
           smtp_from_name: Rados Sanity Cluster Alerts
       desc: Enable email alerts
 
+  - test:
+      name: Configure client admin
+      desc: Configure client on node8
+      module: test_client.py
+      polarion-id:
+      config:
+        command: add
+        id: client.1                      # client Id (<type>.<Id>)
+        node: node8                       # client node
+        install_packages:
+          - ceph-common
+          - ceph-base
+        copy_admin_keyring: true          # Copy admin keyring to node
+        caps:                             # authorize client capabilities
+          mon: "allow *"
+          osd: "allow *"
+          mds: "allow *"
+          mgr: "allow *"
+
   - test:
       name: Testbed setup for RADOS
       module: rados_prep.py
       config:
-        ec_pool:
-          create: true
+# Commenting creation of EC pool as it is not supported in stretch cluster configuration
```

Comment:
Can we avoid leaving code that is commented? Also, it not being supported makes more sense in removing it from here.

Reply:
done